uniswap_universal_router.py

```diff
@@ -191,36 +209,42 @@ def make_trade(self, from_token, to_token, amount, fee, slippage, pool_version="
             slippage (float): Slippage tolerance in percent
         """
         # Convert addresses to checksum format
+        if from_token <= to_token:
+            token0 = from_token
+            token1 = to_token
+            zeroForOne = True
+        else:
+            token0 = to_token
+            token1 = from_token
+            zeroForOne = False
+
+        token0 = Web3.to_checksum_address(token0)
```

Comment:
I'm just using token0 and token1 for v4, it is still using from_token and to_token in v3

Reply:
This is for ordernate the tokens from lowest to highest addreess